src/components/Gnb.js

```diff
@@ -0,0 +1,27 @@
+import logoImg from "../assets/logo.svg";
+import profileImg from "../assets/profile.png";
+
+function Gnb(props) {
```

Comment:
컴포넌트의 props로 받는 데이터는 사실상 무조건 구조분해할당을 해주는게 좋아요~
```suggestion
function Gnb({ hasUserData, email }) {
```